### PR TITLE
Use unsorted arguments by default

### PR DESCRIPTION
**Pull request by 0x0f0f0f**
As discussed in call﻿

### CODE REVIEW

**Comment by 0x0f0f0f** [comment]
@ChrisRackauckas

File: Project.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name = "TermInterface"
 uuid = "8ea1fca8-c5ef-4a55-8b96-4e9afe9c9a3c"
 authors = ["Shashi Gowda <gowda@mit.edu>", "Alessandro Cheli <sudo-woodo3@protonmail.com>"]
-version = "1.0.1"
+version = "1.1.0"
```
Comment (YingboMa) [inline Project.toml:4]: It's breaking.

Comment (0x0f0f0f) [reply Project.toml:4]: 2.0 then?

**Comment by 0x0f0f0f** [comment]
Should we also define `sorted_children` and default `children` to be unsorted? @YingboMa @ChrisRackauckas

**Comment by ChrisRackauckas** [comment]
I'd say yes, @YingboMa ?

**Comment by YingboMa** [comment]
Yeah, I think so too.

**Comment by 0x0f0f0f** [comment]
@ChrisRackauckas @YingboMa  ready for final review

**Review by bowenszhu — APPROVED** [review]

**Review by ChrisRackauckas — APPROVED** [review]
Just waiting on @YingboMa 's opinion now.

**Review by YingboMa — APPROVED** [review]

**Pull request merged**